DROD/EditSelectScreen.cpp

```diff
@@ -2556,7 +2556,7 @@ void CEditSelectScreen::PopulateHoldListBox()
 #ifdef ENABLE_CHEATS
 				false
 #else
-				!g_pTheDB->Holds.PlayerCanEditHold(pHold->dwHoldID)
+				false//!g_pTheDB->Holds.PlayerCanEditHold(pHold->dwHoldID)
```

Comment:
This change doesn't look right.  Maybe a dev tweak that needs to be backed out?

Reply:
Yup, accidentally committed debug code.